### PR TITLE
FORMS-9392: Add vertical tabs styling in WKND theme

### PR DESCRIPTION
**Pull request by akshayvas**
## Screenshots (if appropriate):
vertical-tabs-updated

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: src/site/_variables.scss [hunk 1]
```diff
@@ -43,3 +43,7 @@ $error-outline: 2px $error solid;
 $border-light: 1px $background-color solid;
 $border-error: 1px $error solid;
 $border-dark: 1px $light-gray solid;
+
+$field-margin: .25rem 0;
```
Comment (amit-mnnit) [inline src/site/_variables.scss:47]: Don’t create new variables. please use the existing color code or space variables.

File: src/components/verticaltabs/_verticaltabs.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    font-size: $font-m;
+    margin: $field-margin;
+    position: relative;
+    cursor: pointer;
+  
+  &__tabs-container{
+    display: flex;
+    margin-top: 16px;
+  }
+  &__tabpanel{
```
Comment (amit-mnnit) [inline src/components/verticaltabs/_verticaltabs.scss:11]: please use panel-label mixin here
https://github.com/adobe/aem-forms-theme-wknd/blob/main/src/components/tabsontop/_tabsontop.scss#L6

File: src/components/verticaltabs/_verticaltabs.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    flex: 1;
+    padding: 16px 56px;
+  }
+  &__tablist{
```
Comment (amit-mnnit) [inline src/components/verticaltabs/_verticaltabs.scss:15]: can you create mixins for tablist and tab?
verticalTabList
verticalTab

**Review by amit-mnnit — CHANGES_REQUESTED** [review]
Don’t Create new variables

**Comment by akshayvas** [comment]
@amit-mnnit addressed review comments, please review

**Review by amit-mnnit — APPROVED** [review]

**Pull request merged**